launch/tier4_perception_launch/launch/obstacle_segmentation/ground_segmentation/ground_segmentation.launch.py

```diff
@@ -324,7 +324,7 @@ def create_single_frame_outlier_filter_components(input_topic, output_topic, con
                 ],
                 parameters=[
                     {
-                        "use_lane_filter": False,
+                        "use_lane_filter": True,
```

Comment:
This is debatable. The detection area is looking at pointcloud, but it is often defined outside of ll2 lanes.

Reply:
Even if we use lane filter for inpaint, the elevation value in cells including `min_num_points_per_cell` or more points outside of lanelet is calculated. 
This PR influences only cells outside of lanelet that do not include `min_num_points_per_cell` points.

(And if we can expand Polygon of lanelet, we can include cells in detection_area outside of lanelet.)

Reply:
I see. :+1:  I was mistaken.

Reply:
@yukkysaito Although, setting default to True is not good idea, I think. I will fix this.
And this is a bit confusing, is it better to move these parameters to ground_segmentation.param.yaml? What do you think?